package.json

```diff
@@ -30,6 +30,7 @@
 	},
 	"devDependencies": {
 		"@fluidframework/build-common": "^2.0.3",
+		"copy-webpack-plugin": "^12.0.2",
```

Comment:
Is this dependency necessary?

Reply:
Yes, to copy the image assets to the build output directory. Unless there is a simpler way to do that - this is how we do it elsewhere in the FluidFramework repo.